package.json

```diff
@@ -102,6 +102,7 @@
     "lerna": "^4.0.0",
     "lint-staged": "^12.2.1",
     "mockdate": "^3.0.5",
+    "node-fetch": "2.6.7",
```

Comment:
its overriden in the global setup fiile, and replaced with a mock. This was the quickest patch to fix this, since removing the global mock resulted in alot of failed tests.

Reply:
have you tried `jest.requireActual('fetch')`?

Reply:
Does not work.